### PR TITLE
test: Unit test for logs helper file

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Helps #5320 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
- Testing log information storage for valid and invalid templates

#### Changes proposed in this pull request:
- Unit test for logging helper file

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Don't resolve hound issues unless outdated

**Comment by mrsaicharan1** [comment]
@iamareebjamal Didn't see that one. Will take care.

**Comment by mrsaicharan1** [comment]
@iamareebjamal Please review.

File: tests/all/integration/api/helpers/test_log.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            self.assertTrue(db.session.query(Activity).filter_by(actor=test_actor).first().action,
+                            'User logging@test.com' + ' (' +str(test_user.id)+')'+' created')
+
+    def test_record_activity_invalid_template(self):
+        """Test to record activity for invalid template"""
+        with app.test_request_context():
+            test_user = create_user(email="logging@test.com", password="logpass")
+            record_activity('invalid_template', login_user=test_user, user=test_user)
+            test_actor = test_user.email + ' (' + str(test_user.id) + ')'
+            self.assertTrue(db.session.query(Activity).filter_by(actor=test_actor).first().action, '[ERROR LOGGING] invalid_template')
```
Comment (iamareebjamal) [inline tests/all/integration/api/helpers/test_log.py]: Reverse the arguments

File: tests/all/integration/api/helpers/test_log.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    def setUp(self):
+        self.app = Setup.create_app()
+
+    def test_record_activity_valid_template(self):
+        """Test to record activity for valid template"""
+        with app.test_request_context():
+            test_user = create_user(email="logging@test.com", password="logpass")
+            record_activity('create_user', login_user=test_user, user=test_user)
+            test_actor = test_user.email + ' (' + str(test_user.id) + ')'
+            self.assertTrue(db.session.query(Activity).filter_by(actor=test_actor).first().action,
```
Comment (iamareebjamal) [inline tests/all/integration/api/helpers/test_log.py]: Same

File: tests/all/integration/api/helpers/test_log.py (hunk 1 above)
Comment (mrsaicharan1) [reply tests/all/integration/api/helpers/test_log.py]: Will do it.

File: tests/all/integration/api/helpers/test_log.py (hunk 2 above)
Comment (mrsaicharan1) [reply tests/all/integration/api/helpers/test_log.py]: On it.

File: tests/all/integration/api/helpers/test_log.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+class TestLogging(OpenEventTestCase):
+    def setUp(self):
+        self.app = Setup.create_app()
+
+    def test_record_activity_valid_template(self):
+        """Test to record activity for valid template"""
+        with app.test_request_context():
+            test_user = create_user(email="logging@test.com", password="logpass")
+            record_activity('create_user', login_user=test_user, user=test_user)
+            test_actor = test_user.email + ' (' + str(test_user.id) + ')'
```
Comment (iamareebjamal) [inline tests/all/integration/api/helpers/test_log.py]: Extract `' (' + str(test_user.id) + ')'` to a variable

**Comment by mrsaicharan1** [comment]
👍

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review.

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please review

**Pull request merged**